### PR TITLE
Update to pandoc 1.16

### PR DESCRIPTION
**Pull request by Jmuccigr**

### CODE REVIEW

File: Library/Formula/pandoc.rb [hunk 1]
```diff
@@ -5,8 +5,8 @@ class Pandoc < Formula
 
   desc "Swiss-army knife of markup format conversion"
   homepage "http://pandoc.org"
-  url "https://hackage.haskell.org/package/pandoc-1.15.2.1/pandoc-1.15.2.1.tar.gz"
-  sha256 "60bcb0e65ecb63953dd26d5aacf1a5df83700d116062ffaeffe9edbc9be6df59"
+  url "https://hackage.haskell.org/package/pandoc-1.16/pandoc-1.16.tar.gz"
+  sha256 "f8643d0b67a2de27c860ef79713d078bb1981539c7da30e8aeec2b5f97e8bec4"
```
Comment (mikemcquaid) [inline Library/Formula/pandoc.rb]: Seems wrong: https://travis-ci.org/Homebrew/homebrew/jobs/99983961#L358

**Comment by Jmuccigr** [comment]
Yep, that file is now giving me 

20c0b19a6bf435166da3b9e400c021b90687d8258ad1a0aecfc49fce1f2c6d0c

for the SHA 256 checksum, as the check is showing.

The version I downloaded earlier did not. I'll do another PR.

**Comment by Jmuccigr** [comment]
Added required update to pandoc-citeproc 0.9.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `pandoc 1.16`.

**Pull request closed without merging**